xwalk.gyp

```diff
@@ -632,8 +634,11 @@
       ],
       'copies': [
```

Comment:
Understand now. I noticed if we change the `grit_out_dir`, not only the `*.pak` files are generated in new destination, but also the new generated folder `grit` which contains` xwalk_string.h`.
So let's just keep the old logic to make it easy.